ansible_rulebook/json_generator.py

```diff
@@ -161,6 +161,42 @@ def visit_condition(parsed_condition: ConditionTypes, variables: Dict):
                     ),
                 }
             }
+        elif parsed_condition.operator == "in":
+            return {
+                "ItemInListExpression": {
```

Comment:
Seems all the expressions having lhs and rhs have the same value. Can you extract them to share a function?

Reply:
@bzwei made it into a function call.